modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,176 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, OpenShift Service Mesh (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like OpenShift Monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with User-Workload Monitoring.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Complete one of the following options to configure the Service Mesh control plane for User-Workload Monitoring.
+** Either disable the installation of the default Prometheus instance:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    prometheus:
+      enabled:  false # <1>
+  meshConfig:
+    extensionProviders:
+    - name: prometheus # <2>
+      prometheus: {}
+  security:
+    manageNetworkPolicy: false # <3>
+----
+<1> Disable the default Prometheus instance provided by OSSM.
+<2> Configure the Prometheus extension provider.
+<3> Disable OSSM managed network policies. 
++
+[NOTE]
+====
+This option disables all of your OSSM managed network policies.
+====
++
+** Or add an additional network policy to enable ingress from `user-workload-monitoring`:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: user-workload-access
+  namespace: bookinfo
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          network.openshift.io/policy-group: monitoring
+  podSelector: {}
+  policyTypes:
+  - Ingress
```

Comment:
@fjglira could you modify our e2e test of integration with OpenShift Monitoring stack to use this network policy instead of disabling network policies in SMCP?